BUILD

```diff
@@ -29,3 +29,35 @@ Maven repository for it. To build the binding:
         -Dpackaging=jar
 
 3. Uncomment `<module>nosqldb</module>` and run `mvn clean package`.
+
+# Mapkeeper 
+
+Mapkeeper binding doesn't get built by default because there is no
+Maven repository for it. To build the binding:
+
+1. follow "https://github.com/m1ch1/mapkeeper/wiki/Getting-Started" to Generate Thrift MapKeeper Binding
+   a.  install libevent
+   b.  install boost
+   c.  install thrift
+   d.  generate mapkeeper binding
+
+2. After generating mapkeeper binding, go the "mapkeeper/thrift/gen-java/target", and 
```

Comment:
yes, it uses maven, but now there is no maven repository. Next step, I will try to build a new maven repository for mapkeeper!